### ISSUE TITLE
How did you become a member of the Yeoman team?

### ISSUE THREAD

**Issue by sindresorhus** [SBoudrias/ama#2]
Any tips for people wanting to get involved?

**Comment by SBoudrias** [comment]
> Any tips for people wanting to get involved?

Just do it, there's plenty to do! And we're a good bunch doing useful code review! I learned a LOT working with the Yeoman team.

For me, Yeoman really started out of a sub-project from [backbone-boilerplate](https://github.com/backbone-boilerplate/backbone-boilerplate). We wanted to provide a Yeoman generator to our users so they could consume the boilerplate more easily. So I set to the task!

At the time, there was no [Inquirer](https://github.com/SBoudrias/Inquirer.js) out there. The Yeoman terminal interface was kind of very bad! At the time, I had seen an interactive interface being dynamically refreshed in the Karma cli. The UX was horrible, but I thought the concept was genius. So I set to read Karma cli code, and implemented the same concept of refreshing prompts to create the list prompt we now all know. That prompt was initially bundled inside `generator-bbb`, and soon after the initial generator release, I set on the task of extracting the prompt system to its own module. That's when Inquirer was born. Side note, Inquirer is named in honor of the Citizen Kane movie (Inquirer being Kane's journal name).

At that point, I just suggested Yeoman switch the default prompt system to inquirer. Doing so, I had to dig through the Yeoman code base, and continuing my work on `generator-bbb`, I found a bunch of bugs and issues that I set myself to fix. I think there was just room for a new active maintainer on the core of the Yeoman project, and I just filled the role. I also always liked to design system and framework, so working on the core really was an interesting task for me.

The rest is history!
